C/manual/tips-appendix.docbook

```diff
@@ -6457,4 +6316,662 @@ bogus_symbol2,price2,date2
+                </para>
+              </entry>
+
+              <entry>
+                <para><ulink url="https://www.fundlibrary.com"/>
+                </para>
+
+                <para>This module uses an id that represents the mutual fund on
+                  <ulink url="https://www.fundlibrary.com"/>. There is no easy way of fetching the
+                  id except to jump onto the fundlibrary website, look up the fund and view the url
+                  for clues to its id number.
+                </para>
+              </entry>
+            </row>
+
+            <row>
+              <entry>
+                <para>HElsinki stock eXchange, FI
+                </para>
+              </entry>
+
+              <entry>
+                <para>hex
+                </para>
+              </entry>
+
+              <entry>
+                <para><ulink url="https://www.hex.com"/>
+                </para>
+              </entry>
+            </row>
+
+            <row>
+              <entry>
+                <para>Man Investments, AU
+                </para>
+              </entry>
+
+              <entry>
+                <para>maninv
+                </para>
+              </entry>
+
+              <entry>
+                <para><ulink url="https://www.maninvestments.com.au"/>
+                </para>
+              </entry>
+            </row>
+
+            <row>
+              <entry>
+                <para>Morningstar, SE
+                </para>
+              </entry>
+
+              <entry>
+                <para>morningstar
+                </para>
+              </entry>
+
+              <entry>
+                <para><ulink url="https://www.morningstar.se"/>
+                </para>
+              </entry>
+            </row>
+
+            <row>
+              <entry>
+                <para>Paris Stock Exchange/LeRevenu, FR
+                </para>
+              </entry>
+
+              <entry>
+                <para>lerevenu
+                </para>
+              </entry>
+
+              <entry>
+                <para><ulink url="https://bourse.lerevenu.com"/>
+                </para>
+              </entry>
+            </row>
+
+            <row>
+              <entry>
+                <para>Platinum Asset Management, AU
+                </para>
+              </entry>
+
+              <entry>
+                <para>platinum
+                </para>
+              </entry>
+
+              <entry>
+                <para><ulink url="https://www.platinum.com.au"/>
+                </para>
+              </entry>
+            </row>
+
+            <row>
+              <entry>
+                <para>SIX Swiss Exchange Funds, CH
+                </para>
+              </entry>
+
+              <entry>
+                <para>sixfunds
+                </para>
+              </entry>
+
+              <entry>
+                <para><ulink url="https://www.six-swiss-exchange.com"/>
+                </para>
```

Comment:
This and the next were joined to SIX.
Perhaps do it like in Amsterdam Euronext eXchange, NL?

Reply:
Done